project.md

```diff
@@ -58,11 +74,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_I think most of the time would be spent on the language aspects of this project because there is already a system set in place for javascript, which is what is currently used in this domain. I think figuring out how to implement the language in this environment would be complicated._
```

Comment:
I agree that the language part definitely seems like the most important part since your goal is to replace what already exists.